src/activity_actor.cpp

```diff
@@ -5883,8 +5891,8 @@ void forage_activity_actor::finish( player_activity &act, Character &who )
     // Survival gives a bigger boost, and Perception is leveled a bit.
     // Both survival and perception affect time to forage
 
-    ///\EFFECT_PER slightly increases forage success chance
-    ///\EFFECT_SURVIVAL increases forage success chance
+    /** @EFFECT_PER slightly increases forage success chance */
+    /** @EFFECT_SURVIVAL increases forage success chance */
```

Comment:
The existing codebase is roughly evenly split between `///\` and `/** @` syntax. I chose to standardize on the latter because it avoids the need for blank lines between comments. The prior comments here were not parsed by doxygen as expected, putting the effect_survival line within the doc produced by the effect_per line.